package/wpe/wpeframework-plugins/Config.in

```diff
@@ -26,13 +26,29 @@ config BR2_PACKAGE_WPEFRAMEWORK_COMPOSITOR_OUTOFPROCESS
     help
       Select this to run this plugin in its own process.
 
-config BR2_PACKAGE_WPEFRAMEWORK_COMPOSITOR_NEXUS_SERVER
-    bool "Nexus server"
-    depends on BR2_PACKAGE_HAS_NEXUS
+choice
+    prompt "resolution"
+    depends on !BR2_PACKAGE_WPEFRAMEWORK_COMPOSITOR_NEXUS_SERVER_EXTERNAL
+
+menuconfig BR2_PACKAGE_WPEFRAMEWORK_COMPOSITOR_RESOLUTION_720P
+    bool "720p"
+
+menuconfig BR2_PACKAGE_WPEFRAMEWORK_COMPOSITOR_RESOLUTION_1080P
+    bool "1080p"
+
+menuconfig BR2_PACKAGE_WPEFRAMEWORK_COMPOSITOR_RESOLUTION_2160P
+    bool "2160p"
+
+endchoice
+
+config BR2_PACKAGE_WPEFRAMEWORK_COMPOSITOR_NEXUS_SERVER_EXTERNAL
+    bool "Nexus server external"
+    depends on BR2_PACKAGE_HAS_NEXUS  && !BR2_PACKAGE_WAYLAND
+    default n
     help
-      Include a nxserver with the compositor.
+      Use an external nxserver with the compositor.
 
-if BR2_PACKAGE_WPEFRAMEWORK_COMPOSITOR_NEXUS_SERVER
+if BR2_PACKAGE_HAS_NEXUS && !BR2_PACKAGE_WPEFRAMEWORK_COMPOSITOR_NEXUS_SERVER_EXTERNAL && !BR2_PACKAGE_WAYLAND
```

Comment:
If we are running Westeros/Westong, we still need the NexusServer and unless it is requested to run externally, we can set the configuration so I think that BR2_PACKAGE_HAS_NEXUS && !BR2_PACKAGE_WPEFRAMEWORK_COMPOSITOR_NEXUS_SERVER_EXTERNAL is sufficient. It should not eclude the settings on a chosen (BR2_PACKAGE_WAYLAND)

Reply:
Updated, please review